Class-Code/Game of Life/jkohl/sketch.js

```diff
@@ -1,14 +1,77 @@
-//Your Name(s)
-//Project Name
-//Date
+//Jacob Kohl
+//Game of Life
+//March 15, 2019
 
-/* A brief description of what the program is */
 
+let current = [];
+let cellsize = 10;
+
+let columns,rows;
 
 function setup() {
-    createCanvas(400, 400);
+    createCanvas(900, 500);
+
+    columns = floor(width/cellsize);
+    rows = floor(height/cellsize)
+
+    for(let row = 0; row < rows; row++){
+        current.push(new Array(columns).fill(0).map(()=>round(random())))
```

Comment:
This line is long. Might look better as
```
for(let row = 0; row < rows; row++){
    current.push(
        new Array(columns)
        .fill(0)
        .map(()=>round(random()))
    )
}
```

Reply:
Also, this could use a comment or could be turned into a function for clarity's sake. i.e. what is your intent with this loop